app/src/main/res/layout/fragment_about.xml

```diff
@@ -58,7 +58,8 @@
                     android:layout_marginLeft="@dimen/spacing_negative"
                     android:layout_marginRight="@dimen/spacing_negative"
                     android:layout_marginStart="@dimen/spacing_negative"
-                    android:layout_marginTop="@dimen/spacing" />
+                    android:layout_marginTop="@dimen/spacing"
+                    android:layout_marginBottom="@dimen/spacing_small"/>
 
                 <TextView
                     style="@style/AboutTitle"
```

Comment:
I guess TextView marginTop is better like this.

```xml
<TextView
    style="@style/AboutTitle"
    android:layout_marginTop="@dimen/spacing"
    android:text="@string/about_social" />
```

Of cource both are good, but other almost layouts use `marginTop` to adjust space.